frontend/components/JoinGraveBtn.tsx

```diff
@@ -648,16 +673,59 @@ export default function JoinGraveBtn({
       );
     } else {
       return (
-        <Button
-          onClick={handleJoin}
-          disabled={loading}
-          mb="10px"
-          fontFamily="Bungee"
-          fontSize="16px"
-          fontWeight="400"
-        >
-          {loading ? 'Processing...' : 'START'}
-        </Button>
+        <>
+          <AlertDialog
+            motionPreset="slideInBottom"
+            leastDestructiveRef={cancelRef}
+            onClose={() => {
+              setLoading(false);
+              onClose();
+            }}
+            isOpen={isOpen}
+            isCentered
+          >
+            <AlertDialogOverlay />
+            <AlertDialogContent>
+              <AlertDialogHeader>Found existing URDs</AlertDialogHeader>
+              <AlertDialogCloseButton />
+              <AlertDialogBody>
+                You currently have a Universal Receiver Delegate set for LSP7 (
+                {URDLsp7}) and/or LSP8 ({URDLsp8}). Joining the Grave in this
+                state will overwrite the current URD and will affect
+                functionality of your UP profile that relay on them.
```

Comment:
can we make it with less text or separated in different lines? 
Maybe the addresses can be just a link